includes/class-wc-swedbank-plugin.php

```diff
@@ -0,0 +1,488 @@
+<?php
+
+namespace SwedbankPay\Checkout\WooCommerce;
+
+defined( 'ABSPATH' ) || exit;
+
+use WC_Order;
+use WC_Admin_Meta_Boxes;
+use Exception;
+
+class WC_Swedbank_Plugin {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class WC_Swedbank_Plugin has 17 non-getter- and setter-methods. Consider refactoring WC_Swedbank_Plugin to keep number of methods under 10.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-checkout/pullRequest?prid=5510424)